### PR TITLE
BFS in undirected graph

### PR DESCRIPTION
**Pull request by mahak-bagha**
merge this

### CODE REVIEW

File: BFS/BFS.cpp [hunk 1]
```diff
@@ -0,0 +1,66 @@
+#include <bits/stdc++.h>
+#define ll long long
+
+using namespace std;
+vector<ll> gr[1010];
+
+void bfs(ll u,ll V,bool vis[])
+{
+
+
```
Comment (t2013anurag) [inline BFS/BFS.cpp]: remove extra line

File: BFS/BFS.cpp [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        q.pop();
+        cout<<ver<<" ";
+        for(ll i=0;i<gr[ver].size();i++)
+        if(!vis[gr[ver][i]]){
+                vis[gr[ver][i]]=true;
+                q.push(gr[ver][i]);
+        }
+    }
+
+
```
Comment (t2013anurag) [inline BFS/BFS.cpp]: here also

File: BFS/BFS.cpp [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+}
+void bfsMain(ll src,ll V)
```
Comment (t2013anurag) [inline BFS/BFS.cpp]: extra line

File: BFS/BFS.cpp [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+{
+       bool vis[V];
```
Comment (t2013anurag) [inline BFS/BFS.cpp]: proper indentation

File: BFS/BFS.cpp [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+0 2
+1 3
+2 4
+1 4
+3 4
+3 5
+4 5
+6 7
+0
+*/
```
Comment (t2013anurag) [inline BFS/BFS.cpp]: create a readme file also, explaining what the code does.

File: BFS/BFS.cpp [hunk 6]
```diff
@@ -0,0 +1,66 @@
+#include <bits/stdc++.h>
```
Comment (t2013anurag) [inline BFS/BFS.cpp]: directory name should be small_case.

Comment (mahak-bagha) [reply BFS/BFS.cpp]: changes done @t2013anurag

**Comment by t2013anurag** [comment]
@mahak-bagha  - Can you squash the commits to a single commit?

**Pull request merged**